src/pages/SignTransaction/components/TransactionTokenCard.tsx

```diff
@@ -6,34 +6,42 @@ import {
   Tooltip,
   Typography,
 } from '@avalabs/core-k2-components';
-import { balanceToDisplayValue } from '@avalabs/core-utils-sdk';
 import {
-  TransactionNft,
-  TransactionToken,
-} from '@src/background/services/wallet/handlers/eth_sendTransaction/models';
+  TokenType,
+  type ERC1155Token,
+  type ERC721Token,
+  type NetworkContractToken,
+  type NetworkToken,
+  type TokenDiffItem,
+} from '@avalabs/vm-module-types';
+import { TokenIcon } from '@src/components/common/TokenIcon';
+import { CollectibleMedia } from '@src/pages/Collectibles/components/CollectibleMedia';
 import { useConvertedCurrencyFormatter } from '@src/pages/DeFi/hooks/useConvertedCurrencyFormatter';
-import { BN } from 'bn.js';
 import { useEffect, useRef, useState } from 'react';
-import { useTranslation } from 'react-i18next';
 
 export enum TransactionTokenCardVariant {
   SEND = 'SEND',
   RECEIVE = 'RECEIVE',
   DEFAULT = 'DEFAULT',
 }
 
+const isNftToken = (
```

Comment:
We already have a util for this in `src/background/services/balances/nft/utils/isNFT.ts`

Reply:
This is a different model (not a variation of `TokenWithBalance`)